routes/todoRouter.js

```diff
@@ -0,0 +1,17 @@
+const router = require('express').Router();
+const TodoController = require('../controllers/todoController');
+const authentication = require('../middlewares/authentication');
+const authorization = require('../middlewares/authorization');
+
+router.use(authentication);
+
+router.get('/', TodoController.findAll);
+router.get('/:id', TodoController.findById);
```

Comment:
findById memang sengaja ngga dipakein authorization kah Yan?

Reply:
iya, krn query di controllernya findById nya menggunakan where id & UserId. kalau misalkan ga cocok akan kembaliin Todo not found. daripada kembaliin 'you are not authorized' yang berarti sebenarnya Todo tsb exist tapi emg ga dikasi akses, mending kan sekalian kembaliin 'To Do not found' biar seakan2 Todo tsb memang ga ada. haha